docs/UserGuide.md

```diff
@@ -93,7 +93,7 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to AA.
+Adds a person.
```

Comment:
it's a small detail to remove the 'AA' however it keeps things short and simple for the user to understand :) good job !!